components/ingest-service/backend/elastic/elastic.go

```diff
@@ -38,17 +42,36 @@ func (es *Backend) Initializing() bool {
 	return !es.initialized
 }
 
-func (es *Backend) JobStatus(ctx context.Context, jobID string) (backend.JobStatus, error) {
+func (es *Backend) UpdateProjectTags(ctx context.Context, projectTaggingRules map[string]*iam_v2.ProjectRules) ([]string, error) {
+	logrus.Debug("starting project updater")
+
+	esNodeJobID, err := es.UpdateNodeProjectTags(ctx, projectTaggingRules)
+	if err != nil {
+		return []string{}, errors.Wrap(err, "Failed to start Elasticsearch Node project tags update")
+	}
+
+	esActionJobID, err := es.UpdateActionProjectTags(ctx, projectTaggingRules)
```

Comment:
This adds the actions to the update manager process. https://github.com/chef/automate/issues/62